### PR TITLE
version info in buy and sell tags

### PR DESCRIPTION
**Pull request by mkavinkumar1**
I hope this won't produce much noise as old format is kept for backtesting.

### CODE REVIEW

**Comment by eSeR1805** [comment]
Why is this needed in live/dry? simple tg `/version` tells you what you're running.
Also this will break quick/long sell branches in live/dry

File: NostalgiaForInfinityX.py [hunk 1]
```diff
@@ -8848,65 +8861,65 @@ def custom_sell(self, pair: str, trade: 'Trade', current_time: 'datetime', curre
         if all(c in ['31', '32', '33', '34', '35', '36'] for c in buy_tags):
```
Comment (eSeR1805) [inline NostalgiaForInfinityX.py:8861]: This will not be met in live/dry with the extra version tag in the list.

Comment (mkavinkumar1) [reply NostalgiaForInfinityX.py:8861]: fixed

File: NostalgiaForInfinityX.py [hunk 2]
```diff
@@ -8843,7 +8843,7 @@ def custom_sell(self, pair: str, trade: 'Trade', current_time: 'datetime', curre
         buy_tag = 'empty'
         if hasattr(trade, 'buy_tag') and trade.buy_tag is not None:
             buy_tag = trade.buy_tag
-        buy_tags = buy_tag.split()
+        buy_tags = buy_tag.split('|')
```
Comment (eSeR1805) [inline NostalgiaForInfinityX.py:8846]: This actually breaks it further

**Comment by eSeR1805** [comment]
You've not yet answered the main question here. why is this need?

**Comment by mkavinkumar1** [comment]
> Why is this needed in live/dry? simple tg `/version` tells you what you're running. Also this will break quick/long sell branches in live/dry

by this we can identify the version of NFIX when users share their results in live NFIX channel. without this some peoples thought their bot not taking trades properly.

**Review by eSeR1805 — COMMENTED** [review]
Seems to me like you're not really testing the changes and PR-ing into a codebase used by 100s of people

**Comment by eSeR1805** [comment]
> > Why is this needed in live/dry? simple tg `/version` tells you what you're running. Also this will break quick/long sell branches in live/dry
> 
> by this we can identify the version of NFIX when users share their results in live NFIX channel. without this some peoples thought their bot not taking trades properly.
Having 2 versions showing up in reports will add even more confusion. It will add to the 'what's the best version?' phenomenon which is harmful. Also we're assuming people are roughly using the latest when they post results to the channel. And last but not least it's a development server not a service or helpdesk. People that have issues usually don't bother to read docs to learn the basics of bot usage.
IMO it's extra clutter but I appreciate the contribution.

**Comment by mkavinkumar1** [comment]
> Having 2 versions showing up in reports will add even more confusion. It will add to the 'what's the best version?' phenomenon which is harmful. Also we're assuming people are roughly using the latest when they post results to the channel. And last but not least it's a development server not a service or helpdesk. People that have issues usually don't bother to read docs to learn the basics of bot usage. IMO it's extra clutter but I appreciate the contribution.

No it will show only one version of nfix that you are currently using by that we can clear the confusion for example[https://discord.com/channels/864473885969350666/903273400879968256/942694669509345300]

**Comment by mkavinkumar1** [comment]
> Having 2 versions showing up in reports will add even more confusion. It will add to the 'what's the best version?' phenomenon which is harmful. Also we're assuming people are roughly using the latest when they post results to the channel. And last but not least it's a development server not a service or helpdesk. People that have issues usually don't bother to read docs to learn the basics of bot usage. IMO it's extra clutter but I appreciate the contribution.

By this we can minimize this:
[1.](https://discord.com/channels/864473885969350666/903273400879968256/942378500239806485)
[2](https://discord.com/channels/864473885969350666/903273400879968256/942694669509345300)
[3.](https://discord.com/channels/864473885969350666/903273400879968256/942378130813882379)
[4.](https://discord.com/channels/864473885969350666/887605726669836288/941495309312352306)

**Comment by mkavinkumar1** [comment]
> Having 2 versions showing up in reports will add even more confusion. It will add to the 'what's the best version?' phenomenon which is harmful. Also we're assuming people are roughly using the latest when they post results to the channel. And last but not least it's a development server not a service or helpdesk. People that have issues usually don't bother to read docs to learn the basics of bot usage. IMO it's extra clutter but I appreciate the contribution.

IMO It will provide more clarity

**Review by iterativv — COMMENTED** [review]
We can add this, as extra information, probably it's not very useful but it doesn't do any harm.

**Comment by eSeR1805** [comment]
> also some people thought they have running latest version even when their auto update script broken

Again something that is usage related and that can be simply checked via telegram version function. I'm not in favor of adding clutter for some edge cases.

**Comment by mkavinkumar1** [comment]
> Again something that is usage related and that can be simply checked via telegram version function. I'm not in favor of adding clutter for some edge cases.

that means we have to check version for each commit and auto update script may be failed anytime. Also we no need to ask each person what version they are running like [this](https://discord.com/channels/864473885969350666/903273400879968256/948670873328558090).

**Pull request closed without merging**